tests/storage/cbt/utils.py

```
@@ -59,6 +59,17 @@ def wait_for_vm_cbt_enabled(vm: VirtualMachine) -> None:
             return
 
 
+def wait_for_push_backup_complete(backup: VirtualMachineBackup) -> None:
+    """Wait until a push-mode backup completes successfully."""
+    LOGGER.info(f"Waiting for push-mode backup {backup.name} to complete")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)